package.json

```diff
@@ -93,6 +93,7 @@
     "concurrently": "^8.2.2",
     "config": "^3.3.9",
     "jsbarcode": "^3.11.6",
+    "lucide-react": "^0.525.0",
```

Comment:
I'm not sure if there was a decision in refinement or something about adding this library but assuming there wasn't, we would prefer to just add the BookUser icon as a one off custom icon to reduce library bloat. Have a look at the components/Icons directory for some examples.